File: python/agentic_commerce_protocol/models/checkout_session_update_request.py
# this file is @generated
import typing as t

from .address import Address
from .buyer import Buyer
from .common import BaseModel
from .item import Item


class CheckoutSessionUpdateRequest(BaseModel):
    buyer: t.Optional[Buyer] = None
    items: t.Optional[t.List[Item]] = None
    fulfillment_address: t.Optional[Address] = None
    fulfillment_option_id: t.Optional[str] = None

File: kotlin/src/main/kotlin/com/svix/agentic_commerce_protocol/CheckoutSessionUpdateRequest.kt
// This file is @generated
package com.svix.agentic_commerce_protocol

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CheckoutSessionUpdateRequest(
    val buyer: Buyer? = null,
    val items: List<Item>? = null,
    @SerialName("fulfillment_address") val fulfillmentAddress: Address? = null,
    @SerialName("fulfillment_option_id") val fulfillmentOptionId: String? = null,
)
